dvc/repo/move.py

```diff
@@ -65,4 +65,4 @@ def move(self, from_path, to_path):
 
     stage.dump()
 
-    self.remind_to_git_add()
```

Comment:
Also not using scm_context for move(), so user won't be reminded.

Reply:
As above.

Reply:
As discussed, wrapping with scm_context.